clusterloader2/pkg/measurement/common/bundle/test_metrics.go

```diff
@@ -266,9 +266,10 @@ func execute(m measurement.Measurement, config *measurement.MeasurementConfig) (
 	return m.Execute(config)
 }
 
-func appendResults(summaries *[]measurement.Summary, errList *errors.ErrorList, summaryResults []measurement.Summary, errResult error) {
+func appendResults(summaries *[]measurement.Summary, errList *errors.ErrorList, summaryResults []measurement.Summary,
+	errResult error, measurement string, action string) {
 	if errResult != nil {
-		errList.Append(errResult)
+		errList.Append(fmt.Errorf("action %v failed for %s measurement: %v", action, measurement, errResult))
```

Comment:
NIT: Could you use go-errors, here is an example - https://github.com/kubernetes/perf-tests/blob/75dfad9672dfabe11b292c246f8f99c79db84655/clusterloader2/pkg/config/template_provider.go#L240
They are better than fmt errors, e.g. they add stacktraces

Reply:
sure @mm4tt